src/main/java/org/sonatype/repository/conan/internal/proxy/ConanProxyFacet.java

```diff
@@ -69,12 +75,16 @@
 public class ConanProxyFacet
     extends ProxyFacetSupport
 {
+  private final ConanAbsoluteUrlIndexer absoluteUrlRemover;
```

Comment:
Is it still an `absoluteUrlRemover`?

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/6/commits/705b6cabf74c968309eefed4ba2ac6f6be278ee6